@xen-orchestra/lite/src/stories/web-core/ui/table-pagination/ui-table-pagination.story.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[prop('total-items').required().num(), prop('disabled').bool().widget()]"
```

Comment:
When setting required props, you can also use `preset()` to pass a default value:
```suggestion
    :params="[prop('total-items').required().preset(100).num().widget(), prop('disabled').bool().widget()]"
```